packages/cozy-harvest-lib/src/connections/accounts.js

```diff
@@ -109,14 +109,27 @@ const updateAccount = async (client, account) => {
   return data
 }
 
+/**
+ * Creates or updates account
+ * @param  {Object}  client  CozyClient
+ * @param  {Object}  konnector  io.cozy.konnectors document
+ * @param  {Object}  authData   Account auth attribute
+ */
+const saveAccount = (client, konnector, account = {}) => {
+  return account._id
+    ? updateAccount(client, account)
+    : createAccount(client, konnector, account)
+}
+
 /**
  * Get accounts mutations
  * @param  {Object} client CozyClient
  * @return {Object}        Object containing accounts mutations
  */
 export const accountsMutations = client => ({
   createAccount: createAccount.bind(null, client),
-  updateAccount: updateAccount.bind(null, client)
+  updateAccount: updateAccount.bind(null, client),
+  saveAccount: saveAccount.bind(null, client)
```

Comment:
Why is updateAccount necessary here ? saveAccount already does both creating and saving ?

Reply:
Yep, I first removed them but then I put them back as it was easier to test them (i.e just keep their related tests).